packages/lesswrong/lib/collections/tagFlags/views.ts

```diff
@@ -0,0 +1,15 @@
+import { TagFlags } from './collection';
+import { ensureIndex } from '../../collectionUtils';
+
+TagFlags.addView('allTagFlags', terms => {
+  return {
+    selector: {
+      deleted: false,
+    },
+    options: {
+      sort: {order: 1, name: -1},
+    },
+  };
+});
+
+ensureIndex(TagFlags, {deleted:1, name: 1});
```

Comment:
Should be `{deleted:1, order:1, name:1}` if this is supposed to be for the preceding view. (Though this collection is probably not big enough for it to matter.)